modules/update-best-practices.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-best-practices_{context}"]
+= Best practices for cluster updates
+
+{product-title} is designed to provide a robust update experience that allows clusters to update with minimal disruptions to workloads.
```

Comment:
```suggestion
{product-title} provides a robust update experience that minimizes workload disruptions.
```
Reworded to avoid passive voice and the use of "allow" per ISG:
"Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to…”. Whenever possible, use a direct, user-focused alternative such as “you can use the product to…” or “with this product, users can…”.